pbrd/pbr_zebra.c

```diff
@@ -72,6 +72,8 @@ static int interface_add(int command, struct zclient *zclient,
 	if (!ifp->info)
 		pbr_if_new(ifp);
 
+	pbr_nht_nexthop_interface_update(ifp);
```

Comment:
Shouldn't this commit be broken up into this change( to handle the interface up/down events and look at all nhg's for an ifindex that matches? and then a commit that adds the cli changes?

Reply:
Fair enough, will do that.